.bumpversion.cfg

```diff
@@ -4,20 +4,20 @@ commit = True
 tag = False
 tag_name = {new_version}
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+) (?P<releaseTime>.*)?
-serialize = 
+serialize =
 	{major}.{minor}.{patch}
 	{utcnow:%Y-%m-%dT%H:%M:%SZ}
 
 [bumpversion:file:CHANGES.md]
-search = 
+search =
 	[Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 	------------------------------------------------------------------------------------------------------------------
-replace = 
+replace =
 	[Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 	------------------------------------------------------------------------------------------------------------------
-	
+
 	[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
-	
+
```

Comment:
This will always be changed back and forth. The added spaces are applied by `bump2version` called on each new tag.
Better to have an ignore entry to avoid the noise those changes will generate.

Reply:
Sure, we can add as many files in the ignore list as we want....